github/PullRequest.py

```diff
@@ -409,25 +409,67 @@ def create_comment(self, body, commit_id, path, position):
         """
         return self.create_review_comment(body, commit_id, path, position)
 
-    def create_review_comment(self, body, commit_id, path, position):
+    def create_review_comment(
+        # line replaces deprecated position argument, so we put it between path and side
+        self,
+        body,
+        commit,
+        path,
+        line=None,
+        side=None,
+        start_line=None,
+        start_side=None,
+        in_reply_to=None,
+        subject_type=None,
+        as_suggestion=False,
+    ):
```

Comment:
Looking at `create_review`, unset values should be `github.GithubObject.NotSet`, rather than `None`:
```suggestion
    def create_review_comment(
        # line replaces deprecated position argument, so we put it between path and side
        self,
        body,
        commit,
        path,
        line=github.GithubObject.NotSet,
        side=github.GithubObject.NotSet,
        start_line=github.GithubObject.NotSet,
        start_side=github.GithubObject.NotSet,
        in_reply_to=github.GithubObject.NotSet,
        subject_type=github.GithubObject.NotSet,
        as_suggestion=False,
    ):
```